### DISCUSSION TITLE
File extension

### DISCUSSION THREAD

**Discussion by lmoureaux** [hep-statistics-standard/hep-statistics-standard#22] [Technical]
What should be the recommended file extension for HS3? How should an application distinguish between files formatted in JSON/YAML/TOML/XML/whatever?

For instance, `.hs3`, `.hs3.json` or `.json.hs3`?

**Comment by cburgard** [comment]
So far, we have just used `.json`, but having a custom file extension sounds great to me.
My preferred option would be `.hs3.json`, as this would still allow default-tools for working with json files to recognize this immediately, but if anyone has a different opinion, please make yourself heard!

**Reply by jonas-eschle** [reply]
I agree, keeping the actual format ending is crucial to allow normal tools to open them. Prepending something like hs3 seems also reasonable as it is quite verbose about the content, but no strong opinion here
